lib/resources/projects.js

```diff
@@ -9,13 +9,16 @@
 
 const { Form, Project } = require('../model/frames');
 const { getOrNotFound, getOrReject, reject } = require('../util/promise');
-const { success } = require('../util/http');
+const { isTrue, success } = require('../util/http');
 const { QueryOptions } = require('../util/db');
 const Problem = require('../util/problem');
 
 module.exports = (service, endpoint) => {
-  service.get('/projects', endpoint(({ Projects }, { auth, queryOptions }) =>
-    Projects.getAllByAuth(auth, queryOptions)));
+  // specify ?forms=true to get forms nested within projects
```

Comment:
could be a different query parameter? also had a version where the projects with nested forms was a completely different api endpoint. not sure which way to go except adding it here to an existing endopint via `/v1/projects?someflag=true` means fewer documentation changes.

Reply:
The idea of having a single endpoint makes a lot of sense to me! The response will still be an array of projects, so I think it's a good fit.